int32 rainfall
int32 f1_soil_condition
int32 f2_soil_condition
int32 f3_soil_condition
int32 f4_soil_condition

